fxgl-media/src/main/kotlin/com/almasb/fxgl/audio/AudioPlayer.kt

```diff
@@ -138,4 +145,35 @@ class AudioPlayer : EngineService() {
         music.audio.setLooping(true)
         playMusic(music)
     }
+
+    /**
+     * Pauses all active music if it was currently playing.
+     * Can be restarted by [.resumeMusic] and [.resumeAllMusic]
+     */
+    fun pauseAllMusic() {
+        activeMusic.forEach { pauseMusic(it) }
+    }
+
+    /**
+     * Start playing all active music if its not currently playing.
+     */
+    fun playAllMusic() {
```

Comment:
Very nice, thanks.

Based on the doc of `pauseAllMusic()`, I think  `resumeAllMusic`, which would call `resumeMusic`, is missing.

Reply:
Oops, I accidently left that method out. Fixed it.